docs/packages/pylhcsubmitter/autosix.md

```diff
@@ -0,0 +1,181 @@
+
+The `replace_dict` contains variables for your mask as well as variables for the SixDesk environment.
+See the description of `replace_dict` below.
+
+In any other way, these *special* variables behave like normal variables and can also be inserted in your mask.
+They are also looped over in the same manner as any other variable (if given as a list).
+
+!!! tip "Recommended Reading"
+    It is recommended to have read the section about [job submitter](job_submitter.md) before this one, as these share many similarities and details are explained in the previous page.
+
+??? info "Detailed Arguments of the Script"
+    *--Required--*
+    
+    - **mask** *(PathOrStr)*: Program mask to use
+    
+    - **replace_dict** *(DictAsString)*: Dict with keys of the strings to be replaced in the mask (required) as
+        well as the mask_sixdeskenv and mask_sysenv files in the sixdesk_tools
+        module. Required fields are TURNS, AMPMIN, AMPMAX, AMPSTEP,
+        ANGLES. Optional fields are RESUBMISSION, PLATFORM, LOGLEVEL,
+        FIRSTSEED, LASTSEED, ENERGY, NPAIRS, EMITTANCE, DIMENSIONS, WRITEBINS.
+        These keys can also be used in the mask if needed. The values of this
+        dict are lists of values to replace these or single entries.
+    
+    - **working_directory** *(PathOrStr)*: Directory where data should be put
+    
+    *--Optional--*
+    
+    - **da_turnstep** *(int)*: Step between turns used in DA-vs-Turns plot.
+        default: ``100``
+    
+    - **executable** *(PathOrStr)*: Path to executable.
+        default: ``/afs/cern.ch/user/m/mad/bin/madx``
+    
+    - **apply_mad6t_hacks**: Apply two hacks: Removes '<' in binary call and
+        ignore the check for 'Twiss fail' in the submission file.
+        This hack is needed in case this check greps the wrong lines,
+        e.g. in madx- comments. USE WITH CARE!!
+        action: ``store_true``
+    
+    - **jobid_mask** *(str)*: Mask to name jobs from replace_dict
+    
+    - **python3** *(PathOrStr)*: Path to python to use with sixdb (python3 with requirements
+        installed).
+        default: ``python3``
+    
+    - **python2** *(PathOrStr)*: Path to python to use with run_six.sh (python2 with requirements installed).
+        ONLY THE PATH TO THE DIRECTORY OF THE `python` BINARY IS NEEDED!
+        And it can't be an Anaconda Distribution.
+        default: None (uses the first `python` in path)
+    
+    - **resubmit**: Resubmits if needed.
+        action: ``store_true``
+    
+    - **ssh** *(str)*: Run htcondor from this machine via ssh (needs access to the
+        `working_directory`)
+    
+    - **stop_workspace_init**: Stops the workspace creation before initialization, so one can make
+        manual changes.
+        action: ``store_true``
+    
+    - **unlock**: Forces unlocking of folders.
+        action: ``store_true``
```

Comment:
I would just link to the other doc. If you copy paste here we will have to update always twice and I am very certain that already the first update on either end will forget the other one

Reply:
I don't mind updating the other one and having an admonition linking to that.